.github/ISSUE_TEMPLATE/issue--bug-report.md

```diff
@@ -0,0 +1,97 @@
+---
+name: 'Issue: Bug report'
+about: Let us know about an unexpected error or incorrect behavior
+title: "[BUG]"
+labels: bug
+assignees: ''
```

Comment:
Remove assignees, as the creator may not have the info

Reply:
Removed title and assignees. Labels will be assigned for these.